packages/components-js/src/examples/flyout-example-slotted-grid.html

```diff
@@ -0,0 +1,160 @@
+<p-button type="button" aria="{ 'aria-haspopup': 'dialog' }">Open Flyout</p-button>
+
+<p-flyout open="false" aria="{ 'aria-label': 'Sticky Heading' }">
+  <div slot="header">
+    <p-heading tag="h5" size="large">Sticky Heading</p-heading>
+    <p-text size="small">Sticky header text</p-text>
+  </div>
+  <div class="hero-grid">
```

Comment:
Where are these styles coming from?

Reply:
Yeah, right. This was a remnant.

Reply:
This is just for rendering `:frameworkMarkup` in the Storefront example page.